dbos/context.py

```diff
@@ -64,9 +62,9 @@ def __init__(self) -> None:
         self.function_id: int = -1
         self.in_recovery: bool = False
 
-        self.curr_comm_function_id: int = -1
+        self.curr_step_function_id: int = -1
         self.curr_tx_function_id: int = -1
-        self.sql_session: Optional[Session] = None
+        self.db: Optional[Session] = None
```

Comment:
I'm not a fan of renaming sql_session to db.

Reply:
What do you think would be better? I'm not strongly attached to this change, but `sql_session` is long

Reply:
Yeah I get it, it's short, but it isn't correct, because it isn't the DB, it's a transaction / session within the DB.

transaction, txn, db_session, db_transaction, db_txn, something like that... but we also called it 'sql_session' because session is the SQLAlchemy word and sql_ disambiguates it from HTTP session.

Reply:
I'll just revert the change and we can discuss later